aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java

```diff
@@ -367,16 +370,16 @@ public synchronized void initialize(
     @Override
     public <T extends Model> void query(
             @NonNull Class<T> itemClass,
-            @Nullable QueryPredicate predicate,
+            @NonNull QueryOptions options,
             @NonNull Consumer<Iterator<T>> onSuccess,
             @NonNull Consumer<DataStoreException> onError) {
         Objects.requireNonNull(itemClass);
-        // Objects.requireNonNull(predicate); It is not required!
```

Comment:
Can you uncomment it, and assert the non-null state, now that you have `@NonNull`? Also thank you for making this API improvement. The null-ness of this param has always bothered me.